### PR TITLE
multiline-options: Allow multi_line_timeout  set to a none integer value

### PR DESCRIPTION
**Pull request by HofiOne**
The actual implementation. and validations restrict only the value to be higher than the `follow-freq` (ideally multiplication of it). As the `follow-freq` can be much smaller than one second it makes sense to allow this value to be a non-integer value as well.

Signed-off-by: Hofi <hofione@gmail.com>

### CODE REVIEW

File: modules/affile/affile-grammar.ym [hunk 1]
```diff
@@ -89,6 +89,7 @@ affile_grammar_set_wildcard_file_source_driver(WildcardSourceDriver *sd)
 
 %token KW_FSYNC
 %token KW_FOLLOW_FREQ
+%token KW_MONITOR_FREQ
```
Comment (kovgeri01) [inline modules/affile/affile-grammar.ym:92]: why was this added?

Comment (HofiOne) [reply modules/affile/affile-grammar.ym]: wrong rebase (later change)
